README.md

```diff
@@ -73,12 +73,18 @@ make undeploy
 ```
 
 ## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
+To contribute to MintMaker you need to be part of the [MintMaker Maintainers](https://github.com/orgs/konflux-ci/teams/mintmaker-maintainers) team.
 
 **NOTE:** Run `make help` for more information on all potential `make` targets
 
 More information can be found via the [Kubebuilder Documentation](https://book.kubebuilder.io/introduction.html)
 
+## Additional documentation
+You can find additional documentation here:
+
+* [MintMaker docs](https://redhat-exd-rebuilds.github.io/mintmaker-docs/).
+* Konflux's documenation section about MintMaker: [link](https://gitlab.cee.redhat.com/konflux/docs/users/-/blob/main/topics/mintmaker/user.md#introduction).
```

Comment:
I am not sure we can link to a site that's behind VPN from an open source project, WDYT?

Reply:
Good point. Unfortunately AFAIK there's no upstream version of this. So I'll just remove it.